test/unit/mdc-menu/simple.foundation.test.js

```diff
@@ -116,7 +116,14 @@ test('#init throws error when the necessary DOM is not present', () => {
 testFoundation('#open adds the animation class to start an animation',
   ({foundation, mockAdapter}) => {
     foundation.open();
-    td.verify(mockAdapter.addClass(cssClasses.ANIMATING_OPEN));
+    td.verify(mockAdapter.addClass(cssClasses.ANIMATING_OPEN), {times: 1});
+  });
+
+testFoundation('#open does not add the animation class to start an animation when quickOpen is false',
```

Comment:
Missing a complementary test for `#close`? (Also test both `addClass` and `removeClass` after addressing my comment above.)